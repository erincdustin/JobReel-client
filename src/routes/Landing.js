import React from 'react';
import Description from '../components/Description';

export default class Landing extends React.Component{
  render(){
    return(
      <section className="landing">
        <Description />
      </section>
    )
  }
}