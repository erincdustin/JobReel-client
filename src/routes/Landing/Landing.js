import React from 'react';
import Description from '../../components/Description/Description';

export default class Landing extends React.Component{
  render(){
    return(
      <section className="landing">
        <Description />
      </section>
    )
  }
}