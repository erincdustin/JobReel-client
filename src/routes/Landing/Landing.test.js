import React from 'react'
import ReactDOM from 'react-dom'
import Landing from './Landing'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Landing smoke test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Landing />, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

describe('Landing snapshot test', () => {
  it('renders the UI as expected', () => {
    const wrapper = shallow(<Landing />)
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})