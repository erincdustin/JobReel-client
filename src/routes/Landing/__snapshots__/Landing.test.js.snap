// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing snapshot test renders the UI as expected 1`] = `
<section
  className="landing"
>
  <Description />
</section>
`;
